packages/itwinui-react/src/core/Select/Select.tsx

```diff
@@ -763,3 +774,55 @@ type MultipleSelectButtonProps<T> = {
   selectedItemsRenderer?: (options: SelectOption<T>[]) => JSX.Element;
   tagRenderer: (item: SelectOption<T>) => JSX.Element;
 };
+
+// ----------------------------------------------------------------------------
+
+const SelectListbox = React.forwardRef((props, forwardedRef) => {
+  const {
+    defaultFocusedIndex = 0,
+    autoFocus = true,
+    children: childrenProp,
+    className,
+    ...rest
+  } = props;
+
+  const [focusedIndex, setFocusedIndex] = React.useState(defaultFocusedIndex);
+
+  const autoFocusRef = React.useCallback((element: HTMLElement | null) => {
+    queueMicrotask(() => {
+      const firstFocusable = element?.querySelector(
+        '[tabindex="0"]',
+      ) as HTMLElement | null;
+      firstFocusable?.focus();
+    });
+  }, []);
+
+  const children = React.useMemo(() => {
+    return React.Children.map(childrenProp, (child, index) =>
+      React.isValidElement(child) ? (
+        <CompositeItem
+          key={index}
+          render={child}
+          ref={child.props.ref || (child as any).ref}
+        />
+      ) : (
+        child
+      ),
+    );
+  }, [childrenProp]);
+
+  return (
+    <Composite
+      render={<List as='div' className={cx('iui-menu', className)} />}
+      orientation='vertical'
+      role='listbox'
+      aria-orientation={undefined}
```

Comment:
To confirm, is this needed? If so, why?

Reply:
For some reason, `Composite` automatically adds `aria-orientation="vertical"`, even though it's not needed and doesn't always make sense. So I'm just overriding/unsetting it.

In the future, we might want to create our own `Composite` implementation to avoid such hacks.

Reply:
Ohh, I see. But upon inspecting the DOM, I still see `aria-orientation="vertical"` even with this added line. It only goes away after removing `orientation='vertical'` on L817.

Reply:
Huh, that shouldn't be happening. For now I've removed this line ([`2b5a7ad`](https://github.com/iTwin/iTwinUI/pull/2036/commits/2b5a7adc307b49dad869c39fb06728ce856f6011)), since `aria-orientation` is [valid](https://w3c.github.io/aria/#aria-orientation) on `role=listbox`. But we should be careful with this in the future.